src/main/java/com/newbarams/ajaja/module/plan/dto/BanWordValidationResult.java

```diff
@@ -0,0 +1,23 @@
+package com.newbarams.ajaja.module.plan.dto;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.ahocorasick.trie.Emit;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class BanWordValidationResult {
+	private final boolean existBanWord;
+	private final String originSentence;
+	private final List<Emit> banWordResults;
+
+	public BanWordValidationResult(boolean existBanWord, String originSentence) {
+		this.existBanWord = existBanWord;
+		this.originSentence = originSentence;
+		this.banWordResults = Collections.emptyList();
+	}
```

Comment:
생성자(롬복)를 활용해보는 것은 어떤가요~? (사실 취향 차이긴 합니다 ㅋ)

```suggestion
	public BanWordValidationResult(boolean existBanWord, String originSentence) {
		this(existBanWord, originSentence, Collections.emptyList());
	}
```